README.md

```diff
@@ -96,25 +113,57 @@ fundamental differences in quality and design:
+* **CI friendly** - [Config linting][docs.administration.validating], [dry runs][docs.administration.validating], and [unit tests][docs.guides.unit-testing] make Vector CI friendly.
+* **Custom DNS** - [Custom DNS][docs.global-options#dns_servers] makes service discovery possible.
 * **Hot reload** - [Reload configuration on the fly][docs.process-management#reloading] without disrupting data flow.
-* **Configurable concurrency** - All CPU cores (service) or just one (agent) via the [`--threads` flag][docs.process-management#starting].
-* **Programmable transforms** - [Lua][docs.transforms.lua], [Javascript (coming soon)][urls.pr_721], and [WASM (coming soon)][urls.issue_1802].
-* **Templating** - [Use event fields to create dynamic values][docs.reference.templating], enabling dynamic partitioning and more.
-* **Sink healthchecks** - Protection against downstream disruptions make Vector a good citizen.
-* **Customizable schema** - A schema is not forced on you. [Change Vector's schema][docs.global-options#log_schema] to anything you like.
+* **Optional static binary** - [Optional MUSL static binaries][pages.releases] mean zero required dependencies.
+* **Zero delay start** - Starts and restarts without a delay.
+* **Multi-platform** - [Linux, MacOS, Windows, x86_64, ARM64, and ARMv7][docs.installation].
+* **TLS support** - All relevant Vector components offer TLS options for secure communication.
+
+#### Reliability
+
 * **Decoupled buffer design** - Buffers are coupled with sinks; a bad sink won't bring the entire pipeline to a halt.
-* **Thoughtful disk buffering** - Transfers data to operating system memory to reduce the risk of data loss.
-* **Merge split logs** - Easily merge multi-line logs into one event.
-* **Custom DNS** - [Custom DNS][docs.global-options#dns_servers] makes service discovery possible.
-* **Rate-limited logging** - Vector's internal logging is rate-limited and noise-free.
-* **Optionally fully static binary** - Optional MUSL static binaries mean zero required dependencies.
-* **Thoughtful docs** - [Quality documentation][docs.what-is-vector] that respects your time and reduces communication overhead.
-* **Clear Guarantees** - A [guarantee support matrix][docs.guarantees] helps you make the appropriate tradeoffs.
+* **Fibnacci backoffs with jitter** - Vector's backoff algorithm makes Vector a good citizen during outages.
+* **Back pressure & load shedding** - Buffers can be configured to provide back pressure or shed load.
+* **Memory safety** - Vector is built in Rust and is memory safe, avoiding a large class of memory related errors.
+* **Rate-limited internal logging** - Vector's internal logging is rate-limited avoiding IO saturation when errors occur.
+* **Sink healthchecks** - Healthchecks provide startup safety and prevent deploys with bad configuration.
+* **Durable disk buffering** - Vector uses `leveldb` which transfers data to OSm memory for improved durability.
```

Comment:
I don't know what "transfers data to OSm memory" means. Maybe something like: Vector uses `leveldb` for robust data durability across restarts.